lib/use_packwerk/private.rb

```diff
@@ -427,6 +415,7 @@ def self.create_pack_if_not_exists!(pack_name:, enforce_privacy:, enforce_depend
 
     sig { params(original_package: ParsePackwerk::Package).returns(ParsePackwerk::Package) }
     def self.rewrite_package_with_original_packwerk_values(original_package)
+      ParsePackwerk.bust_cache!
```

Comment:
This is because I added a cache to `ParsePackwerk.all`.

I slightly regret doing that, since it ended up being a bit confusing, and I think I want to change to something closer to what packwerk does later:
https://github.com/rubyatscale/parse_packwerk/issues/9